java/src/main/java/org/vowpalwabbit/spark/VowpalWabbitNative.java

```diff
@@ -96,21 +105,35 @@ public VowpalWabbitNative(String args, byte[] model) {
 
     /**
      * Creates a new VW example associated with this this instance.
+     * 
      * @return new {@code VowpalWabbitExample} object.
      */
     public VowpalWabbitExample createExample() {
         return new VowpalWabbitExample(this.nativePointer, false);
     }
 
     /**
-     * Creates a new empty VW example associated with this this instance.
-     * This is used to mark the end of a multiline example.
+     * Creates a new empty VW example associated with this this instance. This is
+     * used to mark the end of a multiline example.
+     * 
      * @return new {@code VowpalWabbitExample} object.
      */
     public VowpalWabbitExample createEmptyExample() {
         return new VowpalWabbitExample(this.nativePointer, true);
     }
 
+    private static final String[] performanceKeys = new String[] { "numberOfExamplesPerPass", "weightedExampleSum",

```

Comment:
Would it be better for this to be an object? numberOfExamplesPerPass and totalNumberOfFeatures should be integers and being forced into doubles seems odd.

Reply:
reason I switched is that a) I'm forcing into double on spark side anyway b) I should start to deal with exceptions (e.g. when calling/creating java objects). 

Regardless, you have a fair point. I'll change it.